app/themes/default/views/layouts/default.html.slim

```diff
@@ -43,9 +43,12 @@ html
                 ul.dropdown-menu
                   li
                     = link_to(t('user.admin.navbar.account_settings'), edit_user_registration_path)
-                  - if can?(:manage, current_tenant)
+                  - if can?(:manage, :all)
                     li
                       = link_to(t('layout.navbar.admin_panel'), admin_path)
+                  - if can?(:manage, current_tenant)
```

Comment:
I'm afraid the menu is too long, and is kind of repeated. If I'm a system admin, how often would I go to the instance admin?